project/plugins.sbt

```diff
@@ -3,4 +3,5 @@ addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.8")
 addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.4")
 addSbtPlugin("de.heikoseeberger" % "sbt-header" % "5.0.0")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.4")
-addSbtPlugin("com.github.gseitz" % "sbt-release" % "1.0.10")
+addSbtPlugin("com.typesafe.sbt" % "sbt-git" % "1.0.0")
+addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.1.0-M9")

```

Comment:
It's more a client side story :/ I'm not a coursier supported at the moment tbh >_>

Reply:
I decided to include it here because we're using it in Raster Foundry (as of https://github.com/raster-foundry/raster-foundry/pull/4320). We've noticed faster builds because of it's improved caching and parallel dependency resolution. 

I think this project's small size makes it a good candidate for doing a trial run. If things go wrong, it'd be easy to gut it out of the project without any problems.